targets/simple_switch/simple_switch.h

```diff
@@ -111,10 +112,16 @@ class SimpleSwitch : public Switch {
   // returns the number of microseconds elasped since the clock's epoch
   uint64_t get_time_since_epoch_us() const;
 
+  // returns the packet id of most recently received packet. Not thread-safe.
+  static packet_id_t get_packet_id() {
```

Comment:
should this be (packet_id - 1) then to match the comment?

Reply:
Fixed